### PR TITLE
qemu_img_bitmap: new case of qemu-img bitmap

### PR DESCRIPTION
**Pull request by hellohellenmao**
cover all the sub commands of bitmap

Signed-off-by: Tingting Mao <timao@redhat.com>

ID: 2018122

### CODE REVIEW

**Comment by hellohellenmao** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.qemu_img_bitmap.q35: PASS (8.67 s)

**Comment by hellohellenmao** [comment]
depends on avocado-framework/avocado-vt/pull/3267

**Comment by hellohellenmao** [comment]
(1/1) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.qemu_img_bitmap.q35: PASS (8.48 s)

**Comment by hellohellenmao** [comment]
@zhencliu Could you please help to review again? Thanks.

File: qemu/tests/qemu_img_bitmap.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
+import logging
```
Comment (zhencliu) [inline qemu/tests/qemu_img_bitmap.py:1]: remove this, logging will not be used in future, use test.log instead

File: qemu/tests/cfg/qemu_img_bitmap.cfg [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    virt_test_type = qemu
+    type = qemu_img_bitmap
+    kill_vm = yes
+    start_vm = no
+    required_qemu = [4.2.0-24, )
+    images = "bitmap_test top"
+    image_name_bitmap_test = images/bitmap_test
+    image_size_bitmap_test = 1G
+    image_format_bitmap_test = qcow2
+    force_create_image_bitmap_test = yes
```
Comment (zhencliu) [inline qemu/tests/cfg/qemu_img_bitmap.cfg:12]: remove_image_bitmap_test = yes

File: qemu/tests/cfg/qemu_img_bitmap.cfg [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    nbd_image_tag = 'bitmap'
+    enable_nbd_bitmap = yes
+    nbd_port_bitmap = ${nbd_port_top}
+    image_format_bitmap = 'raw'
+    # for merge bitmap
+    image_chain = "bitmap_test top"
+    image_name_top = "images/top"
+    image_size_top = 1G
+    image_format_top = qcow2
+    force_create_image_top = yes
```
Comment (zhencliu) [inline qemu/tests/cfg/qemu_img_bitmap.cfg:28]: remove_image_top = yes

File: qemu/tests/qemu_img_bitmap.py (hunk 1 above)
Comment (hellohellenmao) [reply qemu/tests/qemu_img_bitmap.py]: Updated, thanks.

**Comment by zhencliu** [comment]
LGTM
Let's wait for the dependent patch

**Comment by hellohellenmao** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.qemu_img_bitmap.q35: PASS (9.33 s)

**Comment by hellohellenmao** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.qemu_img_bitmap.q35: PASS (7.10 s)

**Pull request merged**